README.md

```diff
@@ -154,6 +154,7 @@ There's currently documentation for the following:
 - [withFailure](./src/injections/withFailure/)
 - [withPostFailure](./src/injections/withPostFailure/)
 - [withPostSuccess](./src/injections/withPostSuccess/)
+- [withPrefetch](./src/injections/withPrefetch/)
```

Comment:
You should remove `withPrefetch` from the list above this one

Reply:
Also, the function name is `withPreFetch` instead of `withPrefetch`. The folder name has a typo in the code :worried:

Reply:
@mvbattan I think this should be done in another PR. I could do it, though

Reply:
:+1:
You should change the link text so it reads `withPrefetch` anyways, so we show the correct function name in the README